react-portal-employees/src/components/SignUp.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+function SignUp() {
+  return (
+    <div className=''>
+          <div>
+            <h2 className=''>Sign Up </h2> 
+          </div>
+          <form>
+            <div className=''>
+                <label className=''>Login</label>
+                <input type='login'/>
+            </div>
+            <div className=''>
+                <label className=''>Password</label>
+                <input className='' type='password'  />
+            </div>
+            <div className=''>

```

Comment:
why empty className everywhere?

Reply:
I originally did with the style, I forgot to delete, I'll fix it